app/models/user.rb

```diff
@@ -135,6 +135,11 @@ class User < ActiveRecord::Base
   after_create :ensure_in_trust_level_group
   after_create :set_default_categories_preferences
   after_create :set_default_tags_preferences
+  after_create :set_default_sidebar_section_links
+
+  after_update :set_default_sidebar_section_links, if: Proc.new  {
+    self.saved_change_to_staged?
```

Comment:
could you clarify why staged? I would assume that we would like to set default when `active` flag changes to true

Reply:
actually staged may be fine. Potential problem is that `saved_change_to_staged?` is only checking mutation and not value.  Do we need to create defaults only if staged == false to avoid creating defaults for staged users?

That's why I was thinking to switch to active, to be more general, like if user is activated, then default is set

Reply:
The `set_default_sidebar_section_links` will do nothing is `staged == false` so the guard will be done in the other method.